src/glint.gleam

```diff
@@ -886,10 +933,19 @@ fn subcommands_help_to_string(help: List(Metadata), config: Config) -> String {
 
 /// generate the help text for a single subcommand given its name and description
 ///
-fn subcommand_help_to_string(help: Metadata) -> String {
-  case help.description {
-    "" -> help.name
-    _ -> help.name <> "\t\t" <> help.description
+fn subcommand_help_to_string(
+  help: Metadata,
+  longest_subcommand_length: Int,
+) -> String {
+  string.pad_right(help.name, longest_subcommand_length, " ")
+  <> {
+    let description_width =
+      { max_output_width - longest_subcommand_length - tab_width }
+      |> int.max(min_column_width)
+
+    help.description
+    |> utils.wordwrap(description_width)
+    |> string.join("\n\t" <> string.repeat(" ", longest_subcommand_length))
   }
```

Comment:
🔧 considering that the description expression is the majority of the body of this function, we can move it up and bind the description to its own variable to avoid nesting

```suggestion
  let name = string.pad_right(help.name, longest_subcommand_length, " ")
  
  let description_width =
    { max_output_width - longest_subcommand_length - tab_width }
    |> int.max(min_column_width)
  
  let description =
    help.description
    |> utils.wordwrap(description_width)
    |> string.join("\n\t" <> string.repeat(" ", longest_subcommand_length))
  }
  
  name <> description
```